### PR TITLE
Bugfix - Faulty check for both ProviderName and RequesterID

### PR DESCRIPTION
**Pull request by Razumain**
Version 2.0.0 has added a check if both ProviderName and RequesterID are present in the same request. If this is the case, then an error is thrown and the request is refused.

This interpretation has no support from neither SAML nor the eIDAS technical specifications. In fact. Having both these values present in a request is perfectly legal and provides no conflict.

The ProviderName is a "human-readable name of the requester" (as per SAML 2.0). The RequesterID provides zero or more identifiers with the following meaning: "Identifies the set of requesting entities on whose behalf the requester is acting"

IN other words. If the request is sent from the SE connector, then "Sweden eIDAS Connector" is a valid and typical ProviderName. But if that request was originated from service provider A with ID = "https://example.com/provider/a", then a RequesterID would be set to that value.

These attributes have completely different usage and semantics, and in fact, the SE connector sends both of them.

This bug-fix removes the faulty and redundant check both in code and in tests.

### CODE REVIEW

**Comment by bennypi** [comment]
Hi Stefan, 

thanks for your PR.
We have addressed this issue in the latest release 2.1.0 of the eIDAS Middleware and it should be fixed now.

Season's Greetings
Benny
